src/pages/sign-up/index.js

```diff
@@ -4,17 +4,36 @@ import Button from '../../components/button'
 import { useState } from 'react';
 import '../../App.css'
 
-function App() {
+
+
+
+  function App(props) {
   
   const [input, setInput] = useState('')
   const [inputTwo, setInputTwo] = useState('')
+  
+  const onClick = () => {
+    props.history.push('/home')
+  }
+
+  const submitter = () => {
+   return (
+   <div>
+   {input.length===0 || inputTwo.length===0 ? null : onClick() }
```

Comment:
What happens here if I only put put text in the first input?

Reply:
I was a little confused about this actually, at the moment if only one of the inputs is filled the return is still null and nothing happens, both need to be filled to move to the home page, but i expected to need to use && rather than ||